homeassistant/components/rituals_perfume_genie/sensor.py

```diff
@@ -105,7 +105,7 @@ def state(self) -> int:
         return {
             "battery-charge.png": 100,
             "battery-full.png": 100,
-            "battery-75.png": 50,
+            "Battery-75.png": 50,
```

Comment:
Without having actually tried the integration, I would have suspected you needed to rename a png file as well. However I cannot find any of the png files ?

Apart from that simple change.

Reply:
Are we doing web-scraping to get these images?

https://github.com/home-assistant/architecture/blob/master/adr/0004-webscraping.md

Reply:
No, Home Assistant does not use the images we only map a file name to a percentage.
We use the same API as the official app with support from the manufacturer. The API is not public and could change.
The API returns a file name of the image contained in the app based on the current battery level. We could also map the battery description, title, or status id but since the app does not use these we do not do that either, because these could be removed from the API.

Reply:
The API is not public and therefore contains some unusual things like a file name, different use of capital letters, and a file name that does not match the contents of the file.
Anyway, we are very happy that sense company lets us use it and even offers us support.

The image in proposed change is a screenshot of the assets directory of the android APK in android studio, as you can see the file name for `Battery-70.png` has a capital `B` and the others don't.

Reply:
Can we abstract this away from Home Assistant in the pypi so Home Assistant doesn't know about the implementation details?   If they change the images out or the method on how you are detecting the battery percentage, it would be much easier to bump the api then do code review.

Reply:
> Can we abstract this away from Home Assistant in the pypi so Home Assistant doesn't know about the implementation details?

I will do this in another PR along with other parts of the integration. I don't think I can do that for the next release candidate so is it possible to merge this?